src/com/sun/jna/overview.html

```diff
@@ -81,6 +84,52 @@ <h2>Loading JNA</h2>
 library (e.g. for different architectures) in the same directory.
 <p/>
 
+<a href=#navbar_top>Top</a>
+<a name="loading-jdk24"></a>
+<h3>Special considerations for JDK24+</h3>
+The JDK moves to "safe by default" or "Integrity by Default" settings. Settings
+that might compromise the integrity provided by the JVM are off by default.
+Since JDK 24 this also affects the usage of java native interface (JNI). This
```

Comment:
Acronyms are capitalized:

```suggestion
Since JDK 24 this also affects the usage of Java Native Interface (JNI). This
```